hs_core/mongo_discovery_processor.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+from pymongo.mongo_client import MongoClient
+from celery import shared_task
+from datetime import datetime
+
+discovery_url = getattr(settings, "MONGO_DISCOVERY_URL", None)
+db = None
+if discovery_url:
+    client = MongoClient(discovery_url, uuidRepresentation="standard")
+    db = client[getattr(settings, "MONGO_DISCOVERY_DATABASE", "hydroshare_beta")]
```

Comment:
https://pymongo.readthedocs.io/en/stable/api/pymongo/mongo_client.html
I was looking at the docs but didn't understand what the `hydroshare_beta` param is for...can you explain that?

Also, maybe add `MONGO_DISCOVERY_URL` into the settings.py that we have under version control?